test/evmc/evmc.hpp

```diff
@@ -0,0 +1,365 @@
+/* EVMC: Ethereum Client-VM Connector API.
```

Comment:
This is a general comment for all files in the `test/evmc` directory. Could you please update the commit of the evmc code checked in to that used by evmone `v0.1.0` that we are going to be using for testing/fuzzing.

The relevant commit hash of `evmc` is `37a5b8adf650824083c146f229c6426b98a731ea` (see https://github.com/ethereum/evmone/tree/v0.1.0)